onnx2pytorch/convert/layer.py

```diff
@@ -1,11 +1,20 @@
+import numpy as np
 import torch
 from torch import nn
 from onnx import numpy_helper
 
-from onnx2pytorch.operations import BatchNormUnsafe, InstanceNormUnsafe
+from onnx2pytorch.operations import (
+    BatchNormWrapper,
+    InstanceNormWrapper,
+    LSTMWrapper,
+)
 from onnx2pytorch.convert.attribute import extract_attributes, extract_attr_values
 
 
+def _deserialize_to_torch(onnx_param):
```

Comment:
I tried removing the `np.copy` and it turns out our training code using onnx2pytorch still works fine -- I'm guessing there was a bug on our end that's since been fixed. Plus I added a new test `test_train.py` to verify that `np.copy` is unnecessary.